src/client/index.ts

```diff
@@ -110,4 +132,72 @@ export class SocialInboxClient {
   async removeGlobalBlocklist (list: string[]): Promise<void> {
     await this.sendRequest(POST, '/blocklist', TYPE_TEXT, list.join(NEWLINE))
   }
+
+  // Allowlist
+  async getGlobalAllowlist (): Promise<string[]> {
+    const response = await this.sendRequest(GET, '/allowlist')
+    const text = await response.text()
+    return text.split(NEWLINE)
+  }
+
+  async addGlobalAllowlist (accounts: string[]): Promise<void> {
+    await this.sendRequest(POST, '/allowlist', TYPE_TEXT, accounts.join(NEWLINE))
+  }
+
+  async removeGlobalAllowlist (accounts: string[]): Promise<void> {
+    await this.sendRequest(DELETE, '/allowlist', TYPE_TEXT, accounts.join(NEWLINE))
+  }
+
+  // Followers
+  async listFollowers (actor: string): Promise<string[]> {
+    const response = await this.sendRequest(GET, `/${actor}/followers`)
+    return await response.json()
+  }
+
+  async removeFollower (actor: string, follower: string): Promise<void> {
+    await this.sendRequest(DELETE, `/${actor}/followers/${encodeURIComponent(follower)}`)
+  }
+
+  // Hooks
+  async getHook (actor: string, hookType: string): Promise<any> {
+    const response = await this.sendRequest(GET, `/${actor}/hooks/${hookType}`)
+    return await response.json()
+  }
+
+  async setHook (actor: string, hookType: string, url: string, method: 'GET' | 'POST' | 'PUT' | 'DELETE', headers: { [name: string]: string }): Promise<void> {
+    const hook = { url, method, headers }
+    await this.sendRequest(PUT, `/${actor}/hooks/${hookType}`, TYPE_JSON, hook)
+  }
+
+  async deleteHook (actor: string, hookType: string): Promise<void> {
+    await this.sendRequest(DELETE, `/${actor}/hooks/${hookType}`)
+  }
+
+  // Inbox
+  async fetchInbox (actor: string): Promise<any> {
+    const response = await this.sendRequest(GET, `/${actor}/inbox`)
+    return await response.json()
+  }
+
+  async postToInbox (actor: string, activity: APActivity): Promise<void> {
+    await this.sendRequest(POST, `/${actor}/inbox`, TYPE_LDJSON, activity)
+  }
+
+  async approveInboxItem (actor: string, itemId: string): Promise<void> {
+    await this.sendRequest(POST, `/${actor}/inbox/${itemId}`)
+  }
+
+  async rejectInboxItem (actor: string, itemId: string): Promise<void> {
+    await this.sendRequest(DELETE, `/${actor}/inbox/${itemId}`)
+  }
+
+  // Outbox
+  async postToOutbox (actor: string, activity: APActivity): Promise<void> {
+    await this.sendRequest(POST, `/${actor}/outbox`, TYPE_LDJSON, activity)
+  }
+
+  async fetchOutboxItem (actor: string, itemId: string): Promise<APActivity> {
+    const response = await this.sendRequest(GET, `/${actor}/outbox/${itemId}`)
```

Comment:
Instead of having the actor in the params, have it be the last param and set it to `this.account` as a default.

This way only admins would need to specify that param since regular users can only access their own inbox.

The `postToInbox` method should stay the same though.